src/inventory_client/inventory_client.go

```diff
@@ -80,6 +82,11 @@ func CreateInventoryClient(clusterId string, inventoryURL string, pullSecret str
 			RootCAs:            certs,
 		},
 	})
+	// Add retry settings
+	clientConfig.Transport = rehttp.NewTransport(transport,
+		rehttp.RetryMaxRetries(10),
+		rehttp.ExpJitterDelay(2, time.Duration(2)),
```

Comment:
Capped at only 2 nanoseconds?

Reply:
According to https://godoc.org/github.com/PuerkitoBio/rehttp#ExpJitterDelay, it can return a delay between 0 and 2 * 2 ^ 10 (2048) nanoseconds, but no more than 2 nanoseconds. Does it make sense?